direct/src/distributed/AsyncRequest.py

```diff
@@ -1,3 +1,5 @@
+from __future__ import annotations
```

Comment:
Not OK, requires Python 3.7 - would rather have a different solution even if we have to just use Any for this particular annotation